### ISSUE TITLE
Contacts from WSJT-X are not being logged for ARRL-DIGI contest

### ISSUE THREAD

**Issue by ny4i** [n4af/TR4W#597]
ARRL-DIGI is just like WWDIGI. In uWSJT-X, a case statement that set 

```delphi
exch.ExchString := gridSquare;
exch.DomesticQTH := gridSquare;
```
for WWDIGI needs to be updated to also do this for contest ARRL-DIGI.

**Comment by n4af** [comment]
`[Fix 4.111.1](https://tr4w.net/4.111/tr4w_setup_4_111.1.exe)`

**Comment by ny4i** [comment]
Closed with recent build
